nlp_profiler/noun_phase_count.py

```diff
@@ -3,7 +3,7 @@
 from nlp_profiler.constants import NaN
 import re, emoji
 
-def gather_nouns(sentence: str):
+def gather_nouns(sentence):
```

Comment:
Please return the types on the function parameters, this isn't an issue  - please check the build logs of other branches, 3.6 supports it as well